index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+  <head>
+    <meta charset="utf-8">
+    <title>To do List</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <h1>Minha Lista de Tarefas</h1>
+      <p id="funcionamento">Clique duas vezes em um item para marcá-lo como completo</p>
+    </header>
+    <section>
+      <div class="buttom">
+        <button id="apaga-tudo">Limpar Lista</button>
+        <button id="remover-finalizados">Limpar Atividades Finalizadas</button>
+        <button id="salvar-tarefas">Salvar Tarefas</button>
+        <button id="mover-cima">↑</button>
+        <button id="mover-baixo">↓</button>
+        <button id="remover-selecionado">Remover Selecionado</button>
+      </div>
+      <div class="input">
+        <input type="text" id="texto-tarefa">
+        <button id="criar-tarefa">Adicionar</button>
+      </div>
+      <div class="list">
+        <b><h2>lista:</h2></b>
```

Comment:
Talvez esta tag `b` seja redundante, já que o estilo padrão de `h2` já tem um peso de `bold`. Mas se quiser manter por questões semânticas, talvez seja melhor alterar para `strong`. Faz sentido?
```suggestion
       <h2>lista:</h2>
```
```suggestion
        <strong><h2>lista:</h2></strong>
```

Reply:
Verdade não precisava! Retirei aqui.